docs/openj9_support.md

```diff
@@ -186,39 +187,39 @@ When public support for an operating system version ends, OpenJ9 can no longer b
 
 ## Build environments
 
-The project build and test OpenJDK with OpenJ9 on a number of platforms. The operating system and compiler levels for the build systems are shown in the following tables.
+The project builds and tests OpenJDK with OpenJ9 on a number of platforms. The operating system and compiler levels for the build systems are shown in the following tables.
 
 ### OpenJDK 8
 
-| Platform                                    | Operating system          |  Compiler                       |
-|---------------------------------------------|---------------------------|---------------------------------|
-| Linux x86 64-bit                            | CentOS 6.10               | gcc 7.5                         |
-| Linux on POWER&reg; LE 64-bit               | CentOS 7.8                | gcc 7.5                         |
-| Linux on IBM Z&reg; 64-bit                  | RHEL 7.7                  | gcc 7.5                         |
-| Windows x86 32-bit                          | Windows Server 2012 R2    | Microsoft Visual Studio 2013 Update 5 |
-| Windows x86 64-bit                          | Windows Server 2012 R2    | Microsoft Visual Studio 2013 Update 5 |
-| macOS x86 64-bit                            | OSX 10.14.6               | xcode 10.3 and clang 10.0.1     |
-| AIX POWER BE 64-bit                         | AIX 7.1 TL04              | xlc/C++ 13.1.3                  |
+| Platform                      | Operating system       | Compiler                              |
+|-------------------------------|------------------------|---------------------------------------|
+| Linux x86 64-bit              | CentOS 6.10            | gcc 7.5                               |
+| Linux on POWER&reg; LE 64-bit | CentOS 7.8             | gcc 7.5                               |
+| Linux on IBM Z&reg; 64-bit    | RHEL 7.7               | gcc 7.5                               |
+| Windows x86 32-bit            | Windows Server 2012 R2 | Microsoft Visual Studio 2013 Update 5 |
+| Windows x86 64-bit            | Windows Server 2012 R2 | Microsoft Visual Studio 2013 Update 5 |
+| macOS x86 64-bit              | OSX 10.14.6            | xcode 10.3 and clang 10.0.1           |
+| AIX POWER BE 64-bit           | AIX 7.1 TL04           | xlc/C++ 13.1.3                        |
```

Comment:
This should be TL05, for all 8, 11, 16. I expect it's been wrong from a while, TL04 went out of support in 2019.